update/docinfo.xml

```diff
@@ -0,0 +1,12 @@
+<title>Updating</title>
+<productname>{product-title}</productname>
+<productnumber>{product-version}</productnumber>
+<subtitle>Updating OpenShift Service Mesh</subtitle>
+<abstract>
+    <para>This documentation provides information about how to update OpenShift Service Mesh 3.0.
```

Comment:
If you use an attribute for 3.0 here, you won't have to update it in the next version's release I think.

Reply:
done